.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: SDK unit tests
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - 'develop'
+      - 'feature/**'
```

Comment:
- feature-branch is just used sometimes
- I would not add a UMCS- since those branches usually get testet when the pr on develop is done. I thought for feature branches it would be more helpful to have the test run earlier.